openedx/edx-ora2
# make staff column disable on esg enable and show demo on esg enable

**TL;DR -** Make staff column disable on esg enable and show demo on esg enable

JIRA:
- [AU-593](https://openedx.atlassian.net/browse/AU-593)
- [AU-570](https://openedx.atlassian.net/browse/AU-570)

**What changed?**

- ESG flag enabled

<img width="1314" alt="Screen Shot 2022-04-06 at 3 21 26 PM" src="https://user-images.githubusercontent.com/83240113/162201474-9c5a33ce-104c-498d-a313-54afa4521e1d.png">

- ESG flag disabled

<img width="1382" alt="Screen Shot 2022-04-06 at 3 22 39 PM" src="https://user-images.githubusercontent.com/83240113/162201554-08da73f7-921f-48ff-b705-54f3a0af55ce.png">

**Developer Checklist**


**Testing Instructions**

- Enabled ESG
  - Go to [http://localhost:18000/admin/waffle/flag/](http://localhost:18000/admin/waffle/flag/)
  - add/update Flag
    - name `openresponseassessment.enhanced_staff_grader`
    - change everyone to `yes`
    - check `staff`
    - Save
  - [ ] Go to `instructor dashboard -> Open Responses`
  - [ ] staff column should be disabled
  - [ ] `staff_grader` column should show esg link

- Disabled ESG
  - Go to [http://localhost:18000/admin/waffle/flag/](http://localhost:18000/admin/waffle/flag/)
  - add/update Flag
    - name `openresponseassessment.enhanced_staff_grader`
    - change everyone to `no`
    - check `staff`
    - Save
  - [ ] Go to `instructor dashboard -> Open Responses`
  - [ ] staff column should behave like it used to
  - [ ] `staff_grader` column should show esg link to demo

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @openedx/content-aurora
